lib/rabbitmq/cli/plugins/plugins_helpers.ex

```diff
@@ -70,17 +70,44 @@ defmodule RabbitMQ.CLI.Plugins.Helpers do
     end
   end
 
+  @spec enabled_plugins_file(map()) :: {:ok, charlist()} | {:error, any()}
   def enabled_plugins_file(opts) do
     case Config.get_option(:enabled_plugins_file, opts) do
-      nil -> {:error, :no_plugins_file}
-      file -> {:ok, file}
+      nil ->
+        case :os.type do
+          {:unix, _} ->
+            sys_prefix = System.get_env("SYS_PREFIX", "/")
+            config_base_dir = Path.join([sys_prefix, "etc", "rabbitmq"])
+            data_dir = Path.join([sys_prefix, "var", "lib", "rabbitmq"])
+            legacy_location = Path.absname(Path.join([config_base_dir, "enabled_plugins"]))
+            modern_location = Path.absname(Path.join([data_dir, "enabled_plugins"]))
+            case {File.exists?(modern_location), File.exists?(legacy_location)} do
+              {false, true} ->
+                {:ok, legacy_location}
+              _ ->
+                {:ok, modern_location}
+            end
+          {:win32, _} ->
+            rabbitmq_base = case System.fetch_env("RABBITMQ_BASE") do
+                              {:ok, Value} ->
+                                Value
+                              _ ->
+                                app_data = System.get_env("APPDATA")
+                                Path.join([app_data, "RabbitMQ"])
+                            end
+            {:ok, Path.join([rabbitmq_base, "enabled_plugins"])}
+        end
+      file ->
+        {:ok, file}
```

Comment:
The duplication of the logic between rabbitmq-server and this repository makes me think more and more there is something wrong with that file, but at a deeper level. Otherwise the patches solve the immediate issue.